doc/INSTALL_CUSTOM.md

```diff
@@ -59,6 +59,7 @@
 | `usersysgroups` |  |  | Add created user to this additional groups (comma separated and quoted). Non-existent groups will be created as system groups. (e.g. 'usersysgroups=video,www-data') |
 | `user_ssh_pubkey` |  |  | Public SSH key for created user; the public SSH key must be on a single line, enclosed in quotes |
 | `user_is_admin` |  | `0`/`1` | Set to 1 to install sudo and make the user a sudo user. |
+| `rootpw` | raspbian |  | Sets password for root. To disable root completely, also set root_ssh_pubkey empty. |
```

Comment:
The parameter looks **a bit** lost misplaces... But since there is no other setting - it always would.